core/database_arango.py

```diff
@@ -512,14 +555,30 @@ def link_to(
             created=datetime.datetime.now(datetime.timezone.utc),
             modified=datetime.datetime.now(datetime.timezone.utc),
         )
+        print(
+            relationship.source,
+            relationship.target,
+            json.loads(relationship.model_dump_json()),
+        )
```

Comment:
debug print needs to go
```suggestion
```

Reply:
forgot to remove after debugging! we should add more `logging.debug` in database_arango :D

Reply:
Fixed in 5bdc3d53ccaa6a135117ddddccaf78be2c225a3b